Source/Core/CullingVolume.js

```diff
@@ -142,11 +149,14 @@ define([
      *                                 volume, such that if (planeMask & (1 << planeIndex) === 0), for k < 31, then
      *                                 the parent (and therefore this) volume is completely inside plane[planeIndex]
      *                                 and that plane check can be skipped.
+     * @param {Function} [checkClipped] A function which takes the boundingVolume as a parameter, that if supplied,
+     *                                  will check to see if the tile has been entirely clipped by any tileset
+     *                                  clippingPlane.
      * @returns {Number} A plane mask as described above (which can be applied to this boundingVolume's children).
      *
      * @private
      */
-    CullingVolume.prototype.computeVisibilityWithPlaneMask = function(boundingVolume, parentPlaneMask) {
+    CullingVolume.prototype.computeVisibilityWithPlaneMask = function(boundingVolume, parentPlaneMask, checkClipped) {
```

Comment:
This callback doesn't really belong in `CullingVolume`. The logic of whether to proceed with checking the clipping planes should exist in `Cesium3DTile`.